### PR TITLE
[python] Trivial doc-neatens in prep for web publishing

### PR DESCRIPTION
**Pull request by johnkerl**
Context: https://github.com/single-cell-data/TileDB-SOMA/issues/1041

### CODE REVIEW

File: python-spec/src/somacore/data.py [hunk 1]
```diff
@@ -45,6 +45,7 @@ def create(
         *,
         schema: pa.Schema,
         index_column_names: Sequence[str] = (options.SOMA_JOINID,),
+        domain: Optional[Sequence[Optional[Tuple[Any, Any]]]] = None,
```
Comment (atolopko-czi) [inline python-spec/src/somacore/data.py:48]: confused by this diff; all of the `domain` additions here already exists on `main`

Comment (johnkerl) [reply python-spec/src/somacore/data.py:48]: derp my bad i wasn't checked out to `main`

will rebase & repost, sorry

**Comment by johnkerl** [comment]
@atolopko-czi re-ready 🙏

**Review by atolopko-czi — APPROVED** [review]
:shipit:

**Pull request merged**